assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -196,6 +188,18 @@ che.oauth.openshift.verify_token_url= NULL
 # a recurring schedule.
 schedule.core_pool_size=10
 
+# Name of cofig map in Che server namespace with additional CA TLS certificates to be propogated into all user's workspaces.
+# If the property is set on OpenShift 4 infrastructure, then cluster CA bundle will be propogated too.
+che.trusted_ca.bundles_configmap=NULL
```

Comment:
Instead of `che.trusted_ca.` I would recommend consider: `che.infra.kubernetes.` or`che.workspace.trusted_ca.` as configuration domain

And config properties then:
`{DOMAIN}.configmap` <- config map in Che Server namespace to configure workspace trust stores.
`{DOMAIN}.dest_configmap` <- Is managed by Che Server. Admin needs to configure it only in case it conflicts with the already existing config map.
`{DOMAIN}.mount_path`

Reply:
I'm not decided yet. So how the properties should look like?

Reply:
I'm a bit more for `infra.kubernetes` since we are configuring configmap which is stick to k8s. So, then as an example `che.infra.kubernetes.trusted_ca.configmap`
^ if we don't take info account that we may have different infras from k8s or OS, then I'm fully OK with `che.workspace.trusted_ca.configmap`, and maybe even I like it more because it's simpler.

Reply:
The question is between `che.infra.kubernetes.trusted_ca.configmap` and `che.workspace.trusted_ca.configmap`. The same for the other properties.

Reply:
I think, I'll go with `che.workspace.trusted_ca.*`

Reply:
I have changed my mind and decided to go with `che.infra.kubernetes.trusted_ca.*` because we have infra specific labels.